aboutpage.html

```diff
@@ -0,0 +1,89 @@
+                <button>AL Campaign</button>
+              </div>
+                <div class="nav-menu">
+                  <button id="menu" class="drop-button">
+                    <img class="menu-icon" src="./images/nav-menu.png" alt="nav-menu">
+                  </button>
+                  <div class="drop-content" id="drop-content">
+                    <a class="cls-btn"><img src="./images/Icon.png" height="12" width="12" alt="x"></a>
+                    <a href="index.html">Home</a>
+                    <a href="#logo">Logo</a>
+                    <a href="#partner">Partner</a>                   
+                  </div>
+                </div>
+            </nav>
+        </header>
+          
+          <section class="hello-section-about">
+            <p class="hello-p-about">Hello Legends!</p>
+            <div class="hello-cont-about">
+              <h1>APEX LEGENDS WEBINAR IS COMING 24 SEPT 2022</h1>
+              <div class="orange-line-about"></div>
+                <p class="black-p-about">In this webinar, you will learn all about the Apex Legends from the best players, 
+                who will share their mechanics, macro game, and teamwork knowledge about the game.</p>                  
+                    <p class="ea-p-about">Please contact us per Email for any further questions about Apex Legends Webinar!</p>
+                    <p class="ea-email">EA@e.ea.com</p>
+                </div>
+          </section>
+        </div>
+        <div id="logo" class="apex-logo-section">
+            <h2>Apex Legends Webinar 2022 Logo</h2>
+            <div class="orange-line-about"></div>
+            <p>The logo of Apex Legends was decided by Respawn Entertainment.</p>
+            <img class="apex-logo" src="./images/apex-logo1.png" alt="apex-logo">
+        </div>
+        <div class="separator-line"></div>
+        <div class="submit-section">
+          <h2>See the past Apex Legends Webinar Summits</h2>
+          <div class="orange-line-about"></div>
+          <p>Take a look at the last two Apex Legends Global Series winners teams.</p>
+          <div class="tournament-cards">
+          <div class="tournament1">
+            <h3>2022</h3>
+            <p>Apex Legends Global Series</p>
+          </div>
+          <div class="tournament2">
+            <h3>2020</h3>
+            <p>Apex Legends Global Series</p>
+          </div>      
+          </div>
+      </div>
+      
+      <section id="partner" class="partner-section">
+        <div class="partner-title">
+        <h3>Partner</h3>
+        <div class="partner-line"></div>
+      </div>
+        <img class="corsair" src="./images/corsair.png" alt="">
+        <img class="logi" src="./images/logi.png" alt="">
+        <img class="razer" src="./images/razer.svg" alt="">        
+      </section>
+
+      <footer>
+        <img src="./images/apex-logo1.png" alt="apex-logo">
+        <p>2022 Apex Legends USA. Some rights reserved.</p>
+      </footer>
```

Comment:
- [ ] There are some differences between the desk version and the mobile version especially on the about page, kindly correct the partner and footer sections 👇🏽 
- **Deck version**
![image](https://user-images.githubusercontent.com/58642949/191652332-4d7fa2cf-6088-4ba5-ab91-c2b694ba07a0.png)

- **Mobile version**
![image](https://user-images.githubusercontent.com/58642949/191652372-f925c4a8-abe4-4622-9b0f-e987d2175cc4.png)